apps/els_lsp/src/els_log_notification.erl

```diff
@@ -19,7 +19,7 @@ log(#{level := Level} = LogEvent, _Config) ->
     try
         Msg = logger_formatter:format( LogEvent
                                      , #{ template => ?LSP_LOG_FORMAT}),
-        els_server:send_notification(<<"window/logMessage">>, #{
+        els_server:send_log(<<"window/logMessage">>, #{
```

Comment:
A _notification_ is a LSP concept (as opposed to a _request_), so I would prefer to maintain the current naming here. Instead, I would add a dedicated clause to the `do_send_notification/3` for the `window/logMessage` method which skips logging. WDYT?

Reply:
Makes sense, but I'll include a comment (sort of like `filtered out to avoid recurvise logging`) so it makes sense why this specific notification is treated differently

Reply:
Great idea! Feel free to reference the issue number in the comment.

Reply:
Done